spec/models/course_spec.rb

```diff
@@ -2104,4 +2104,50 @@
       )
     end
   end
+
+  describe "#age_minimum" do
+    context "when age_range_in_years set" do
+      it "returns lower age_range_in_years bound" do
+        expect(subject.age_minimum).to eql(3)
+      end
+    end
+
+    context "when age_range_in_years not set" do
+      subject { Course.new }
+
+      it "returns nil" do
+        expect(subject.age_minimum).to be_nil
+      end
+    end
+  end
+
+  describe "#age_maximum" do
+    context "when age_range_in_years set" do
+      it "returns upper age_range_in_years bound" do
+        expect(subject.age_maximum).to eql(7)
+      end
+    end
+
+    context "when age_range_in_years not set" do
+      subject { Course.new }
+
+      it "returns nil" do
+        expect(subject.age_maximum).to be_nil
+      end
+    end
+  end
+
+  describe "#bursary_requirements" do
+    it "returns default requirements" do
```

Comment:
No the default should be blank,

If there is an bursary amount then there is a requirement
If the subject is 03 the both degree and A-level is required


and it is related to 
https://github.com/DFE-Digital/teacher-training-api/blob/dcde01c32c6cd10c879fd747d213274e944af103/app/models/course.rb#L455-L457



![image](https://user-images.githubusercontent.com/470137/89435652-1267f080-d73d-11ea-8a34-f9676ae779fd.png)
[see subject filter](https://www.find-postgraduate-teacher-training.service.gov.uk/results/filter/subject?fulltime=false&hasvacancies=true&parttime=false&qualifications%5B%5D=QtsOnly&qualifications%5B%5D=PgdePgceWithQts&qualifications%5B%5D=Other&senCourses=false)

![image](https://user-images.githubusercontent.com/470137/89435856-5b1fa980-d73d-11ea-89b8-2f9ebe628bc6.png)
[see biology course](https://www.find-postgraduate-teacher-training.service.gov.uk/course/2CG/3BYM)

Reply:
I made some changes, can you double check to make sure I've understood what you said

Reply:
:heart: